index.html

```diff
@@ -50,7 +53,158 @@ <h2 class="greeting greeting-text headline-box">
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">Multi-Post Stories <br />Gain+Glory</h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+      </article>
+
+      <!--About me section-->
+      <article class="about-me">
```

Comment:
You will agree also that this part of the website where you talk about yourself would rather be called a section than an article. So, a `<section>` tag will be more semantic here. Don't you think so?